backend/src/main/java/ch/nova_omnia/lernello/model/data/File.java

```diff
@@ -0,0 +1,36 @@
+package ch.nova_omnia.lernello.model.data;
+
+import java.util.UUID;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "files")
+@Data
+@NoArgsConstructor
+public class File {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "uuid", nullable = false)
+    @NotNull
+    private UUID uuid;
+
+    @Column(name = "name", nullable = false)
+    @NotBlank
+    @Size(min = 1, max = 255)
+    private String name;
+
+    public File(String name) {
```

Comment:
@hajir3  has a way to do this constructor using lombok I think, but this works too